connect/runtime/src/main/java/org/apache/kafka/connect/runtime/distributed/IncrementalCooperativeAssignor.java

```diff
@@ -489,12 +473,10 @@ protected void handleLostAssignments(ConnectorsAndTasks lostAssignments,
                 log.debug("Delayed rebalance in progress. Task reassignment is postponed. New computed rebalance delay: {}", delay);
             } else {
                 // This means scheduledRebalance == 0
-                // We could also also extract the current minimum delay from the group, to make
-                // independent of consecutive leader failures, but this optimization is skipped
-                // at the moment
+                // We could also extract the current minimum delay from the group, to make
+                // independent of consecutive leader failures, but this optimization is skipped at the moment
                 delay = maxDelay;
-                log.debug("Resetting rebalance delay to the max: {}. scheduledRebalance: {} now: {} diff scheduledRebalance - now: {}",
-                        delay, scheduledRebalance, now, scheduledRebalance - now);
```

Comment:
As the comment said, this else block means `scheduledRebalance == 0`, so we don't need to log the `scheduledRebalance` and `now` value